### PR TITLE
adds camoskins to the game and contraband cargo

### PR DESCRIPTION
**Pull request by bigfatbananacyclops**
## About The Pull Request
hello, i took some time making the sprites, they aren't great but i believe they look nice, in the PR it adds 7 camouflage sets, which comes in the fatigues, bulletproof helmet, armor and caps. they are obtainable through contraband cargo, the crates come extras, russian balaclava, combat tackler gloves, assault belt and combat boots.  each crate should come with two sets to equip two players, the cost is 4000 creds.

the camos that are added are the following:

classic woodland.
image

artic camo for killing space commies and chilling in icebox
image

desert camo for invading space iraq?
image

jungle camo to defend against the menace of jungle fever and gorillas
image

there also is some ss13 themed camos, some of them which are kinda funny, lavaland.

image

floortile camo, (this one only comes with a jumpsuit)

image

and lastly a dark camo for dark areas like maintenance.

image

this should be all of it (im aware of the grey camo issue, its been fixed)
image

i probably will add one last camo which would be leopard/animal.

## Why It's Good For The Game
there was absolutely no reason to make this, but i think it would be nice to have it, more cool syndicate clothes for fashion. also more military content which haves no place in a space station.

## Changelog
:cl:
add: 7 camouflage sets of fatigue/bpvest/bphelmet
add: contraband cargo containing this added sets
tweak: some of the contents in the cargo crates + the price
imageadd: camo icons
code: changed pack code
refactor: refactored some code
/:cl:

### CODE REVIEW

**Comment by saprasam** [comment]
GOOD

**Comment by MrDoomBringer** [comment]
haters will call it bloat
those camo patterns look sick good job

**Comment by ShibaInuLord** [comment]
floortile camo...

**Comment by necromanceranne** [comment]
Okay, you sold me at floortile camo.

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 1]
```diff
@@ -1160,3 +1160,29 @@
 /datum/species/golem/mhydrogen/on_species_loss(mob/living/carbon/C)
 	REMOVE_TRAIT(C, TRAIT_ANTIMAGIC, SPECIES_TRAIT)
 	return ..()
+
+/datum/species/golem/pizza
```
Comment (Fikou) [inline code/modules/mob/living/carbon/human/species_types/golems.dm:1164]: ok

Comment (bigfatbananacyclops) [reply code/modules/mob/living/carbon/human/species_types/golems.dm:1164]: wait what, i have that on a different branch

**Comment by optimumtact** [comment]
I think you got lost on the way to that ww2 server

File: html/changelogs/.all_changelog.yml [hunk 2]
```diff
@@ -40451,3 +40451,100 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
   - rscadd: New alien larva spawn tips!
   - tweak: Alien Queens are now easier to distinguish over hivemind communication
       similar to a Blob Overmind.
+2020-07-07:
```
Comment (optimumtact) [inline html/changelogs/.all_changelog.yml:40454]: you've made a real mess of this branch

**Comment by trexter555** [comment]
epic and based

**Comment by bigfatbananacyclops** [comment]
i need to fix this mess

**Pull request closed without merging**